test-apps/display-performance-test-app/src/frontend/DisplayPerformanceTestApp.ts

```diff
@@ -1032,6 +1032,9 @@ async function reopenImodel(testConfig: DefaultConfigs): Promise<void> {
 async function openImodelAndLoadExtViews(testConfig: DefaultConfigs, extViews?: any[]): Promise<void> {
   activeViewState = new SimpleViewState();
 
+  // Do this to prevent view size from changing
+  activeViewState.viewState; // eslint-disable-line @typescript-eslint/no-unused-expressions

```

Comment:
How does this prevent view size from changing?

Reply:
I have no idea, honestly, but it seems to.

Reply:
That was in there before, and I took it out, because it didn't seem like it was doing anything, but putting it back in seemed to fix the scaling problem that David mentioned in his email. He hasn't confirmed that it's fixed for him, yet, but I managed to reproduce the problem and then have it fixed when I added it.

Reply:
I know, also doing a console log that printed out the view.height & view.width seemed to fix the problem too--which also doesn't seem to make sense to me.

Reply:
I'm not a fan of magic. Would be good to understand what's happening.

Reply:
Heather found and fixed the problem, no longer magic.